tests/unit/test_open_telemetry.py

```diff
@@ -109,366 +98,6 @@ def minus_udf(x: int, y: int) -> int:
     dict_exporter.clear()
 
 
-def test_register_udaf_from_file(dict_exporter):
-    test_file = os.path.normpath(
-        os.path.join(
-            os.path.dirname(__file__),
```

Comment:
it seems we removed all those tests? do we have a plan to add it back? or those tests are not necessary? 
Also, curious, what is causing the flakyness?

Reply:
I put a comment above, all these tests are covered in integ tests, so it should be safe to remove them

Reply:
The flakiness is caused by race condition I believe. When testing opentelemetry, I had to initialize a exporter. The exporter is like a global resource that cannot be re-initialized. Which makes all the tests accessing one resource. I am still looking if there is a better way to solve this issue

Reply:
I see. could you update the pr description with what are we doing in this pr, and why the test could be removed safely?